<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>911</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="filter_box">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>760</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FIR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Median</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moving Average</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>760</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load .wav</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filter_freq_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>760</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Frequency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filter_time_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>760</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Time</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reflect_freq_box">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>790</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflect Frequency</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reflect_time_box">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>790</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflect Time</string>
    </property>
   </widget>
   <widget class="QSlider" name="median_filter_slider">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>860</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>103</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="moving_average_slider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>860</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>103</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>840</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>840</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Moving Average Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>840</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FIR Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>790</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="cutoff_input">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>890</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>890</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Cutoff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>860</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Tap count</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pass_zero_checkbox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>860</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass zero</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="taps_input">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>860</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
